CHANGES

```diff
@@ -1,7 +1,7 @@
 DD MMM YYYY - 2.9.2 - To be released
 ------------------------------------
 
- * {dis|en}able-filename-logging: Option to disable logging of stopwatches
```

Comment:
Same as others: make sure to not perform a fix on the top of your own fix. Try to keep this feature a one-patch thing.